pkg/proxy/ipvs/graceful_termination.go

```diff
@@ -80,6 +80,14 @@ func (q *graceTerminateRSList) remove(rs *listItem) bool {
 	return false
 }
 
+// return the size of the list
```

Comment:
this is needed for the test, we have to wait until we have some elements we can delete, and the check of the len has to be locked

Reply:
we really need this function? let me see, Is there another way?